docs/readmes/orc8r/dev_minikube.md

```diff
@@ -157,17 +158,16 @@ kubectl exec -it --namespace orc8r deploy/orc8r-orchestrator -- \
 Now ensure the API and your certs are working
 
 ```bash
-# Tab 1
+# Tab 1 (should hang)
 kubectl --namespace orc8r port-forward svc/orc8r-nginx-proxy 7443:8443 7444:8444 9443:443
 
 # Tab 2
-# Assumes CERTS_DIR is set
+export CERTS_DIR=${MAGMA_ROOT}/.cache/test_certs  # mirrored from above
 curl \
   --insecure \
   --cert ${CERTS_DIR}/admin_operator.pem \
   --key ${CERTS_DIR}/admin_operator.key.pem \
   https://localhost:9443
-
```

Comment:
Nit: I like keeping this newline for visual separation

Reply:
The extra newline makes it seem like the entire area `# Should output: "hello"` rather than just tab 2? 

I'm down to change though! But just some thoughts :)

Reply:
Removing newline works for me